exercises/test/command.py

```diff
@@ -503,7 +506,22 @@ def launch_container_monitor(containers_to_monitor, stop_attached_container):
     """
     monitor_thread = threading.Thread(target=monitor_containers, args=(containers_to_monitor, stop_attached_container), daemon=True)
     dtslogger.info("Starting monitor thread")
-    dtslogger.info(f"Containers to monitor: {[container.name for container in containers_to_monitor]}")
+    dtslogger.info(f"Containedef convertNotebook(filepath, export_path) -> bool:
```

Comment:
Did you forget to close the quotation ? Also, not sure you need a f-string here.

Reply:
I am going to fix that. I don't know why but probably I did `CRTL+V` (?) in a completely wrong place....